dartagnan/src/main/java/com/dat3m/dartagnan/verification/model/event/FenceModel.java

```diff
@@ -0,0 +1,20 @@
+package com.dat3m.dartagnan.verification.model.event;
+
+import com.dat3m.dartagnan.program.event.Event;
+import com.dat3m.dartagnan.program.event.Tag;
+import com.dat3m.dartagnan.verification.model.ThreadModel;
+
+
+public class FenceModel extends DefaultEventModel {
```

Comment:
We should probably also add fences to the witness.

Reply:
I think `Fences` as visible events are already there.